pkg/monitor/monitorapi/identification_pod.go

```diff
@@ -74,6 +74,13 @@ func (r ContainerReference) ToLocator() string {
 }
 
 func ReasonFrom(message string) string {
+	return AnnotationFrom(message, "reason")
+}
+
+// AnnotationFrom extracts annotations of the format key/value from the message/locator string.
+// i.e. "disruption/cache-kube-api connection/new stopped responding to GET requests over new connections"
+//   has annotations "disruption" and "connection"
+func AnnotationFrom(message, annotationName string) string {
```

Comment:
even though the change you made here is equivalent to what was there before (because `"reason"` is passed on line 77 and `annotations["reason"]` is returned, I think line 93 should say `return annotations[annotationName]`.

Reply:
Total bug, thanks!